aliases.js

```diff
@@ -22,7 +22,16 @@ const path = require('path');
 
 const ALIASES = {
   'loaders.gl/test': path.resolve(__dirname, './test'),
-  'loaders.gl': path.resolve(__dirname, './modules/core/src')
+  'test-data': path.resolve(__dirname, './test/data'),
+  '@loaders.gl/core': path.resolve(__dirname, './modules/core/src'),
+  '@loaders.gl/draco': path.resolve(__dirname, './modules/draco/src'),
+  '@loaders.gl/glb': path.resolve(__dirname, './modules/glb/src'),
+  '@loaders.gl/gltf': path.resolve(__dirname, './modules/gltf/src'),
```

Comment:
As proposed in the RFC, I think it makes sense to group GLB with glTF. The glTF piece itself is pretty small, we can't do a whole lot with glTF without actually using a WebGL library (which will happen in luma.gl `glTFInstantiator`), so the overhead is pretty small, and GLB is a part of the glTF standard..